web/main.js

```diff
@@ -124,6 +123,26 @@ class Upload {
         }
     }
 }
+
+//This function is pending to be wired to the file download
+function download() {
+    console.log("DOWNLOADING");
```

Comment:
What's missing here?

Reply:
Since I wasn't able to get the app running correctly, I don't know which of the functions in the file are being called when downloading the file, that's why I left this functionality pending to be wired to the main wormhole download flow.